CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @CermakM @jan-zabloudil @bolshakoff
```

Comment:
@bolshakoff I've invited you to be a collaborator, once you accept, error is fixed

Reply:
Accepted the invitation. 🙏